src/index.js

```diff
@@ -30,6 +30,8 @@ class Sentry {
             method: method,
             headers: Object.assign({}, requestHeaders, headers || {}),
             body: body,
+	    retries: 5,
```

Comment:
It would be nice if these are configurable by the user of this plugin. Aka:

```
retries: this.options.retries || 5,
retryDetail: this.options.timeout || 1000,
```